jib-build-plan/src/test/java/com/google/cloud/tools/jib/api/buildplan/AbsoluteUnixPathTest.java

```diff
@@ -75,13 +76,14 @@ public void testResolve_relativeUnixPath() {
 
   @Test
   public void testResolve_Path_notRelative() {
+    AbsoluteUnixPath absoluteUnixPath = AbsoluteUnixPath.get("/");
+    Path path = Paths.get("/not/relative");
     try {
-      AbsoluteUnixPath.get("/").resolve(Paths.get("/not/relative"));
+      absoluteUnixPath.resolve(path);
```

Comment:
How about we replace the use of the try-catch here with the following?:
```
IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> absoluteUnixPath.resolve(path));
assertThat(exception).hasMessageThat().isEqualTo("Cannot resolve against absolute Path: " + path, ex.getMessage());
```

Reply:
Ah, I like that `assertThrows` avoids the try catch block. Let me update with it.